pkgs/applications/misc/sfwbar/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk3
+, meson
+, ninja
+, json_c
+, pkg-config
+, gtk-layer-shell
+, libpulseaudio
+, libmpdclient
+, libxkbcommon
+,
+}:
+stdenv.mkDerivation rec {
+  pname = "sfwbar";
+  version = "1.0_beta11";
```

Comment:
Is there any particular reason for using this beta release?

Reply:
Yes, there are no proper stable releases worth using - I believe the maintainer treats beta releases akin to actual releases.

Reply:
Ah, got it. Feel free to mark this as resolved; I was just curious.